src/renderer/constants/action_types.js

```diff
@@ -36,6 +37,7 @@ export const SKIP_UPGRADE = 'SKIP_UPGRADE';
 export const START_UPGRADE = 'START_UPGRADE';
 export const AUTO_UPDATE_DECLINED = 'AUTO_UPDATE_DECLINED';
 export const AUTO_UPDATE_DOWNLOADED = 'AUTO_UPDATE_DOWNLOADED';
+export const CLEAR_UPGRADE_TIMER = 'CLEAR_UPGRADE_TIMER';
```

Comment:
This was added to the lbry-redux action types but it should have never been there. I will do some cleanup in the future to make sure we aren't keeping types there that only belong in the app